stores/chroma/README.md

```diff
@@ -24,18 +24,26 @@ const vectorStore = new ChromaVector({
 // Create a new collection
 await vectorStore.createIndex('my-collection', 1536, 'cosine');
 
-// Add vectors
+// Add vectors with documents
 const vectors = [[0.1, 0.2, ...], [0.3, 0.4, ...]];
 const metadata = [{ text: 'doc1' }, { text: 'doc2' }];
-const ids = await vectorStore.upsert('my-collection', vectors, metadata);
+const documents = ['full text 1', 'full text 2'];
+const ids = await vectorStore.upsert(
+  'my-collection',
+  vectors,
+  metadata,
+  undefined, // auto-generate IDs
+  documents  // store original text
+);
 
-// Query vectors
+// Query vectors with document filtering
 const results = await vectorStore.query(
   'my-collection',
   [0.1, 0.2, ...],
   10, // topK
-  { text: { $eq: 'doc1' } }, // optional filter
-  false // includeVector
+  { text: { $eq: 'doc1' } }, // metadata filter
+  false, // includeVector
+  { $contains: 'specific text' } // document content filter
```

Comment:
Have we thought about using an object with params instead of positional args? `{ includeVector: false, $contains: '' }` or does it need to be this way for compatibility reasons?

Reply:
i'm thinking about switching to an object with params. Right now it's for consistency with all the vector dbs, but we should probably change this for all of them to be object params

Reply:
I''ll make a separate PR that handles that